docs/advanced-topics/stale-index.md

```diff
@@ -0,0 +1,172 @@
+# Permanent stale index inconsistency
+
+In some situations due to crashes of storage database, index database, or JanusGraph instances 
+permanent stale index may appear.  
+One of such cases could be a vertex removal from the graph but due to the crash during index persistence 
+there is a chance that index won't be updated ever. In such situation `IllegalStateException` will be 
```

Comment:
`In such situation` is a bit ambiguous. Users might wrongly think that this exception is thrown during "the crash during index persistence". Providing an example of when this exception will be thrown will be super helpful.

Reply:
Will add it tomorrow

Reply:
Tried to rephrase this sentence and add an example. Here is the diff: https://github.com/JanusGraph/janusgraph/compare/13ada03a882b447f813586cabfbda41ca033c0b5..1811ea4cb33fb1c415a4884fa51ec1a111127fb3